src/layout/section/cardsGridSection/CardsGridSection.style.tsx

```diff
@@ -0,0 +1,102 @@
+import styled from 'styled-components'
+
+import { color, componentSizes, responsiveBreakpoints, space } from '../../../_utils/branding'
+
+const wrapperHorizontalPadding = space.xl
+const cardsPerRowSmall = 2
+const cardsPerRowLarge = 3
+
+export const StyledCardsGridSection = styled.div`
+  & .kirk-cardsGridSection-title {
+    margin: ${space.l} 0 ${space.m};
+  }
+
+  & .kirk-cardsGridSection-button {
+    text-align: center;
+  }
+
+  & .kirk-cardsGridSection {
+    margin: auto;
+    display: flex;
+    justify-content: start;
+    padding: ${space.l} 0;
+    box-sizing: border-box;
+    overflow: auto; /* Make Cards scrollable horizontally */
+    -ms-overflow-style: none; /* Remove scrollbar visually IE 10+ */
+    scrollbar-width: none; /* Remove scrollbar visually Firefox */
+    scroll-snap-type: x mandatory;
+    scroll-padding: ${space.xl};
+  }
+
+  & .kirk-cardsGridSection .kirk-card {
```

Comment:
If Card style needs to be redefined, it rather use styled-components syntaxe to avoid classes conflict

```suggestion
import TripCard from "../tripCard"
const Card = styled(TripCard)`
// redefined styles goes here
`
```

Reply:
TripCard items are passed through the children prop so I am not sure I can have them replaced by a styled version.

Reply:
Indeed you would to export the styled version instead of the normal one and apply it